### PR TITLE
[PORT] - Kiss Update (Throw Kiss emote)

### PR DESCRIPTION
**Pull request by ariaworld**
## About The Pull Request

-Makes it so you can throw kisses, just like on TG-Code. ("Inspired" by https://github.com/tgstation/tgstation/pull/56698 and https://github.com/tgstation/tgstation/pull/56732)
NOTE: Does not include the kiss of death lipstick.

## Why It's Good For The Game
Meant to port a really cute addition from TG, (could potentially be used as a base for original RP/ERP additions, for example smooch lipstick marks on mobs).

## Changelog
:cl:
add: Throwing kisses with the *kiss emote. (Original emote moved to *kiss2)
/:cl:

### CODE REVIEW

**Comment by SandPoot** [comment]
I dunno how to feel about keeping the flavor version. Plus failing checks.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -520,10 +520,26 @@
...
+
+	if(new_style && apply_trait)
+		ADD_TRAIT(src, apply_trait, LIPSTICK_TRAIT)
+		hopefully_a_head?.stored_lipstick_trait = apply_trait
+
 /mob/living/carbon/human/clean_blood()
 	var/mob/living/carbon/human/H = src
 	if(H.gloves)
-		if(H.gloves.clean_blood())
+		if(H.gloves.clean_blood())
```
Comment (SandPoot) [inline code/modules/mob/living/carbon/human/human.dm]:
```suggestion
		if(H.gloves.clean_blood())
```
Why do coders do this.

Comment (ariaworld) [reply code/modules/mob/living/carbon/human/human.dm]: It was me from 2 months ago

**Review by deathride58 — APPROVED** [review]

**Comment by ariaworld** [comment]
Bump

**Pull request closed without merging**